### PR TITLE
Legacy corteza server add beta column modification

### PR DESCRIPTION
**Pull request by skamensky**
Unfortunately when doing some cleanup the PR's I created were deleted (probably because I deleted my fork and created a new one).

As I left it standing I needed to clean up some tests that were failing. I'll get working on that now.

### CODE REVIEW

**Comment by skamensky** [comment]
Hey @darh ,

I'm ready for a final review when you guys have the time. I also caught a small bug and fixed it ( see 561a7761a9deed5f018facb679b7195428292442 ) so that a test would not fail.
Some tests do fail, but these were failiing when I started working on this:


`TestApigwRouteUpdate`
`TestAuthExternalSAMLSuccess`
`Test0001_basics_detect_datarace_issues`

**Comment by darh** [comment]
Reviewing this now. It will not go to 2022.9.3 (this week) because I want it to live in a version branch for a couple of days.

**Comment by skamensky** [comment]
What is a version branch? `2022.9.3` is not a version branch?

**Comment by darh** [comment]
version branch would be `2022.9.x`

More info: https://docs.cortezaproject.org/corteza-docs/2022.9/developer-guide/release-cycle/index.html

File: server/pkg/dal/diff.go [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
 )
 
 const (
-	AttributeMissing              modelDiffType = "attributeMissing"
-	AttributeTypeMissmatch        modelDiffType = "typeMissmatch"
-	AttributeSensitivityMissmatch modelDiffType = "sensitivityMissmatch"
+	AttributeMissing             modelDiffType     = "attributeMissing"
+	AttributeTypeMissmatch       modelDiffType     = "typeMissmatch"
+	AttributeSensitivityMismatch modelDiffType     = "sensitivityMismatch"
+	AttributeCodecMismatch       modelDiffType     = "sensitivityMismatch"
```
Comment (darh) [inline server/pkg/dal/diff.go]: value probably needs to be "codecMismatch"

File: server/store/adapters/rdbms/dal/connection.go [hunk 2]
```diff
@@ -161,14 +162,21 @@ func (c *connection) CreateModel(ctx context.Context, mm ...*dal.Model) (err err
 	c.mux.Lock()
 	defer c.mux.Unlock()
 	for _, m := range mm {
-		err = ddl.UpdateModel(ctx, c.dataDefiner, m)
+
+		if err != nil {
+			return
+		}
+		_, err = c.dataDefiner.TableLookup(ctx, m.Ident)
```
Comment (darh) [inline server/store/adapters/rdbms/dal/connection.go:165]: This does not make sense. Probably error check after TableLookup call?

**Review by darh — CHANGES_REQUESTED** [review]
Tried it. Works 🚀 . Two minor things -- see coments.

File: server/store/adapters/rdbms/dal/connection.go (hunk 2 above)
Comment (skamensky) [reply server/store/adapters/rdbms/dal/connection.go:165]: Yeah you're right. There already is an error check after the TableLookup call. The check you noticed was an unnecessary no-op. Fixed.

File: server/pkg/dal/diff.go (hunk 1 above)
Comment (skamensky) [reply server/pkg/dal/diff.go]: Good catch. Fixed.

**Comment by skamensky** [comment]
@darh ,

I've address your comments

**Comment by darh** [comment]
👍 I'll squash and merge in the next couple of days.

**Pull request merged**